elliott/elliottlib/bzutil.py

```diff
@@ -29,9 +29,29 @@
 from elliottlib.metadata import Metadata
 from elliottlib.util import isolate_timestamp_in_release, chunk
 
+
 logger = logutil.get_logger(__name__)
 
 
+def get_jira_client() -> JIRA:
```

Comment:
Why have this? We already have JiraBugTracker class to encapsulate all of the Jira logic